mmv1/third_party/terraform/services/storage/resource_storage_bucket_object.go

```diff
@@ -117,10 +117,9 @@ func ResourceStorageBucketObject() *schema.Resource {
 
 			// Detect changes to local file or changes made outside of Terraform to the file stored on the server.
 			"detect_md5hash": {
-				Type: schema.TypeString,
-				// This field is not Computed because it needs to trigger a diff.
+				Type:     schema.TypeString,
+				Computed: true,
 				Optional: true,
-				ForceNew: true,
```

Comment:
is this field meant to be filled in by api? Based off the comment it seems like this wouldn't be filled out by the user. Which in that case you can just set as output only

Reply:
no this field was there from beginning, from what I did understood it is there to compare change of the bucket object content when mainly a `source` attribut is used. So it is purely internal provider code.

I do not know how to make it as output only, do you have a documentation on this I did not found anything :/ (from the provider doc I seen that `effective_labels` are output-only but looking into code that declare it I did not see any attribute to make it output-only

PS : I did revert the commit 
I look into the test I fix, to make it work I did type mistake.